test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -1183,6 +1200,26 @@ Describe "Invoke-WebRequest tests" -Tags "Feature" {
 
     #endregion charset encoding tests
 
+    #region Client Certificate Authentication
```

Comment:
It is better use `Context {}`. I suggest don't mix HTTP and HTTPS tests and put HTTPS test in separate `Context {}` while we use different tools for them.